typescript/src/processors/readTypeScriptModules/index.ts

```diff
@@ -20,8 +13,12 @@ import { ParameterDoc } from '../../api-doc-types/ParameterDoc';
 import { PropertyMemberDoc } from '../../api-doc-types/PropertyMemberDoc';
 import { TypeAliasExportDoc } from '../../api-doc-types/TypeAliasExportDoc';
 
+import { getExportDocType, ModuleSymbols, TsParser } from '../../services/TsParser';
 import { expandSourceFiles, SourcePattern } from './SourcePattern';
 
+// This import lacks of type definitions.
```

Comment:
lacks "a" type definition?

Reply:
Not too sure to be honest. I think usually an import includes multiple type definitions. Depends on how we interprete "type definition". Can change if preferred though.

Reply:
I mean that "lacks of type definitions" does not read well in English. It should be

```
// This import lacks type definitions.
```

or 
```
// This import lacks a type definition.
```

Reply:
Ah got it. Sorry, didn't see that it doesn't read well. Changed.